Kickstarter-iOS/Views/Controllers/RewardPledgeViewControllerTests.swift

```diff
@@ -13,6 +13,7 @@ internal final class RewardPledgeViewControllerTests: TestCase {
     |> Project.lens.state .~ .live
   fileprivate let cosmicReward = Project.cosmicSurgery.rewards.last!
     |> Reward.lens.shipping.enabled .~ true
+    |> Reward.lens.shipping.summary .~ "Wherever"
```

Comment:
A good example of where multiple optional fields on a struct/product type leads to gotchas! Screenshots rendered storyboard template data since the data came back `nil`. Unlikely event in production, but possible, especially with a backend change!